extras/beast/unit_test/amount.hpp

```diff
@@ -23,29 +23,29 @@ class amount
     std::string const& what_;
 
 public:
-    amount (amount const&) = default;
-    amount& operator= (amount const&) = delete;
+    amount(amount const&) = default;
+    amount& operator=(amount const&) = delete;
 
     template<class = void>
-    amount (std::size_t n, std::string const& what);
+    amount(std::size_t n, std::string const& what);
 
     friend
     std::ostream&
-    operator<< (std::ostream& s, amount const& t);
+    operator<<(std::ostream& s, amount const& t);
 };
 
 template<class>
-amount::amount (std::size_t n, std::string const& what)
-    : n_ (n)
-    , what_ (what)
+amount::amount(std::size_t n, std::string const& what)
+    : n_(n)
+    , what_(what)
```

Comment:
I don't want to be on the cutting edge of that, as compiler support is largely untested.